eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/persistence/SubmittedVariantAccessioningDatabaseService.java

```diff
@@ -30,8 +29,7 @@
 import java.util.List;
 
 public class SubmittedVariantAccessioningDatabaseService
-        extends BasicSpringDataRepositoryDatabaseService<ISubmittedVariant, Long, SubmittedVariantEntity>
-        implements MonotonicDatabaseService<ISubmittedVariant, String> {
+        extends BasicSpringDataRepositoryMonotonicDatabaseService<ISubmittedVariant, SubmittedVariantEntity> {
 
```

Comment:
in this class the method getAccessionInRanges should not be implemented, to use the parent one